packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -318,7 +319,7 @@ export interface IContextualMenuItem {
    * Additional styles to apply to the menu item
    * @defaultvalue undefined
    */
-  style?: React.CSSProperties;
+  style?: IMenuItemStyle;
```

Comment:
This is a breaking change unfortunately, so we have to be careful with it. I don't know what the right approach here would be, since we should definitely push people to the MergeStyles based styling instead of inline React styles. It would also be confusing to have a style and styles prop that do different things. Thoughts @dzearing ?